src/components/Categories/Categories.jsx

```diff
@@ -0,0 +1,28 @@
+import { Link } from 'react-router-dom';
+import { useFilters } from '../../context/filter-context';
+import './Categories.css'
+const Categories = (props) => {
+    const {category} = props;
+    const { productListDispatch } = useFilters()
+    return (
+        <div className="category">   
+            {category.map((c) => (
```

Comment:
I would suggest you replace `category` with `categories` and `c` with `category`

Reply:
Thanks for suggestion